frontend/src/views/WheresMyGeneV2/components/GeneSearchBar/components/ShareButton/utils.tsx

```diff
@@ -87,20 +87,105 @@ export const loadStateFromQueryParams = ({
 }): LoadStateFromURLPayload | null => {
   if (isSSR()) return null;
 
-  const paramsToRemove = [];
+  const paramsToRemove: string[] = [];
 
   // Checks if the URL has any query params related to shared filter state
   // If so, it will update the state with the values from the URL
 
-  // Check for version
+  const { delimiter } = getMetadataForShareLinkParsing({
```

Comment:
create a new utility function `getMetadataForShareLinkParsing` for centralizing metadata used for parsing share link. Right now it just returns `delimiter`, but we could expose `version` and anything else in the future